### DISCUSSION TITLE
How to merge teams

### DISCUSSION THREAD

**Discussion by luckyyangrun** [PaddlePaddle/PaddleSpatial#27] [BaiduKDDCup2022] [answered]
Dear Organizer：
Whether teams are allowed to merge in this competition？How to merge teams?

**Comment by zhoujingbo** [comment, marked as answer]
Hi,

Our system does not support merging teams. Your friends can register with another account to join other teams.
If they do not use the dismissed team to submit any model, we will not consider this case disqualified.

Thanks a lot
